sources/packages/web/src/components/layouts/BodyHeaderContainer.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
May the template not be simplified as below or something similar?
```vue
<template>
  <v-card v-if="card">
    <slot name="header"></slot>
    <slot></slot>
  </v-card>
  <div v-else>
    <slot name="header"></slot>
    <slot></slot>
  </div>
</template>
```

Reply:
I have made the suggested change. There are two slight modifications. 
1) I have wrapped the card / no-card inside a div to give it a `class="mb-5"` to add `bottom-margin` after every card / no-card element. 
2) I have added a `v-container` inside the v-card

Reply:
In terms of code syntax, I would still prefer as below for better readability.
```vue
<template>
  <div class="mb-5">
    <v-card v-if="enableCardView">
      <v-container>
        <slot name="header"></slot>
        <slot></slot>
      </v-container>
    </v-card>
    <div v-else>
      <slot name="header"></slot>
      <slot></slot>
    </div>
  </div>
</template>
```